shell/chart/gatekeeper.vue

```diff
@@ -21,6 +22,8 @@ export default {
   },
 
   computed: {
+    ...mapGetters(['currentCluster']),

```

Comment:
I updated this component to fix a duplicated computed property; the PR was otherwise good.